### PR TITLE
Add account_space_update lambda handler

### PR DESCRIPTION
**Pull request by liam-lloyd**
Since uploads tend to come in large bursts with spans of low activity in between, we want to move asynchronous upload handling to lambdas, so we can scale our resources according to present necessity. To that end, this commit adds a package to stela containing a lambda handler that will update the amount of storage an account has left whenever a new file is stored in our systems.

### CODE REVIEW

**Comment by liam-lloyd** [comment]
To test locally:
0. Choose a record in your local database and delete its corresponding `ledger_nonfinancial` entry
1. Run `docker build --platform linux/amd64 -t account_space_updater:test -f Dockerfile.account_space_updater .`
2. Run `docker run --platform linux/amd64 -p 9001:8080 --env DATABASE_URL=postgres://postgres:permanent@database:5432/permanent account_space_updater:test`
3. Run `docker ps` and note the name of the container you just started running
4. Run `docker network connect devenv_default <YOUR CONTAINER NAME>`
5. Run 
```
curl "http://localhost:9001/2015-03-31/functions/function/invocations" -d '{"Records": [{
                                                                           "type" : "Notification",
                                                                           "messageId" : "55b53693-2003-564a-bdf4-6879e5ca3e3f",
                                                                           "topicArn" : "arn:aws:sns:us-west-2:364159549467:Local_Events_Liam",
                                                                           "body" : "{\"recordId\": \"<YOUR RECORD ID>\"}",
                                                                           "timestamp" : "2024-06-17T23:15:21.000Z",
                                                                           "signatureVersion" : "1",
                                                                           "signature" : "jp52rCPBi2AEQVmj8INqRJulDHgI6C1XuHzvpvv4NeM5P0u2dxJZIb2XWR5+nLBgvx9Ki9JcgOwjvHQjK1b5u/qESRUj1wlMJcdd3FEL+/Yo5cpIP9dER2YDa1kZ5MfG2ccnIiHocETc0locCqurMHoyp3a1CSEE42Nad+qO56i9uUDCKgVvNPGBmYeHJRNW+OsosIBP8F/9a6MnqTOOBSfs8nLGsF1coiQDbVwaqC2scNAHO8rj7s2aVePf/RUopZZbOno/4tZd+0E5G28L+rDiEwpZkKeWE+Sjbeg9R/DYqbvEvRdNm9tfS19xI3d/5xGqE6+H/qtqJjDIEvHq9g==",
                                                                           "signingCertURL" : "https://sns.us-west-2.amazonaws.com/SimpleNotificationService-60eadc530605d63b8e62a523676ef735.pem",
                                                                           "unsubscribeURL" : "https://sns.us-west-2.amazonaws.com/?Action=Unsubscribe&SubscriptionArn=arn:aws:sns:us-west-2:364159549467:Local_Events_Liam:14123677-3ae4-45bb-b82d-7944e1dc8387",
                                                                           "messageAttributes" : {
                                                                             "Entity" : {"stringValue":"record"},
                                                                             "Action" : {"stringValue":"submit"}
                                                                           }
                                                                         }]}'
```
6. Observe that `account_space` and `ledger_nonfinancial` have been updated appropriately
7. Run the same `curl` command and observe that no duplicate changes to `account_space` or `ledger_nonfinancial` occur

File: packages/account_space_updater/src/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+          {
+            messageId: "1",
+            receiptHandle: "1",
+            body: JSON.stringify({ recordId: "1" }),
+            attributes: {
+              ApproximateReceiveCount: "1",
+              SentTimestamp: "1",
+              SenderId: "1",
+              ApproximateFirstReceiveTimestamp: "1",
+            },
```
Comment (cecilia-donnelly) [inline packages/account_space_updater/src/index.test.ts:69]: What are these attributes?

**Review by cecilia-donnelly — COMMENTED** [review]
Thanks for this! I'm not sure how to test it and have one other question - though I expect we'll discuss tomorrow.

**Comment by cecilia-donnelly** [comment]
Sorry, somehow totally didn't see the comment about testing locally just now!

**Review by cecilia-donnelly — APPROVED** [review]
Finally tested! This is exciting; thanks @liam-lloyd .

I noticed that you have a "Liam Local" SNS queue - should we all have those?

**Pull request merged**